### PR TITLE
[Json/Tlv]Add convertTlvTag function

### PR DESCRIPTION
**Pull request by yunhanw-google**
Add convertTlvTag function for android controller use later.
The existing test cover it.

### CODE REVIEW

File: src/lib/support/jsontlv/JsonToTlv.h [hunk 1]
```diff
@@ -32,4 +32,10 @@ CHIP_ERROR JsonToTlv(const std::string & jsonString, MutableByteSpan & tlv);
  */
 CHIP_ERROR JsonToTlv(const std::string & jsonString, TLV::TLVWriter & writer);
 
+/*
```
Comment (yufengwangca) [inline src/lib/support/jsontlv/JsonToTlv.h:35]:
```suggestion
/*
 * Given a uint64_t tagNumber, transform it into a tlv tag. When tagNumber is less than UINT8_MAX, encode the tag using ContextTag, when tagNumber is
 * less than uint32_t, encode the tag using an implicit profile tag.
 */
```

**Review by yufengwangca — APPROVED** [review]

File: src/lib/support/jsontlv/JsonToTlv.h [hunk 2]
```diff
@@ -32,4 +32,13 @@ CHIP_ERROR JsonToTlv(const std::string & jsonString, MutableByteSpan & tlv);
...
+ * Given a uint64_t tagNumber, transform it into a tlv tag. When tagNumber is less than UINT8_MAX, encode the tag using ContextTag, when tagNumber is
```
Comment (saurabhst) [inline src/lib/support/jsontlv/JsonToTlv.h:36]:
```suggestion
 /*
 * Convert a uint64_t tagNumber to a TLV tag. When tagNumber is less than UINT8_MAX,
 * the tag is encoded using ContextTag. When tagNumber is less than UINT32_MAX,
 * the tag is encoded using an implicit profile tag.
 */
```

File: src/lib/support/jsontlv/JsonToTlv.h [hunk 3]
```diff
@@ -32,4 +32,13 @@ CHIP_ERROR JsonToTlv(const std::string & jsonString, MutableByteSpan & tlv);
...
+ * less than uint32_t, encode the tag using an implicit profile tag.
+ */
+ * Given a uint64_t tagNumber, and convert it a tlv tag. when tagNumber is less than UINT8_MAX, tag is encoded with ContextTag, when tagNumber is
```
Comment (saurabhst) [inline src/lib/support/jsontlv/JsonToTlv.h:39]: this can be removed now

Comment (yunhanw-google) [reply src/lib/support/jsontlv/JsonToTlv.h]: done, thanks

**Review by saurabhst — APPROVED** [review]

**Pull request merged**